app/views/flow/index.html.erb

```diff
@@ -47,7 +47,7 @@
     <td style="text-align: right;">
       <%= '%s - <b>%s</b>'.html_safe % [pluralize(order.line_items.sum(:quantity), 'item'), order.flow_total] %>
       <br/>
-      <small><%= order.flow_experience.key %></small>
+      <small><%= order.flow_order ? order.flow_order.experience[:key] : Flow.base_country %></small>
```

Comment:
this is the view that will soon be removed, in our /admin/flow to see all recent orders with status.

the case is, if order is handled by Flow -> get country from experience, but if it is without flow than it is Solidus default -> base_country.

that was my logic. I remember you said we will replace this view with with direct api call